doc/api/net.md

```diff
@@ -214,60 +258,29 @@ net.createServer().listen(
     path.join('\\\\?\\pipe', process.cwd(), 'myctl'))
 ```
 
-The parameter `backlog` behaves the same as in
-[`server.listen([port][, hostname][, backlog][, callback])`][`server.listen(port, host, backlog, callback)`].
-
-*Note*: The `server.listen()` method may be called multiple times. Each
-subsequent call will *re-open* the server using the provided options.
-
-### server.listen([port][, hostname][, backlog][, callback])
+#### server.listen([port][, host][, backlog][, callback])
 <!-- YAML
 added: v0.1.90
 -->
+* `port` {Number}
+* `host` {String}
+* `backlog` {Number} Common parameter of [`server.listen()`][] functions
+* `callback` {Function} Common parameter of [`server.listen()`][] functions
 
-Begin accepting connections on the specified `port` and `hostname`. If the
-`hostname` is omitted, the server will accept connections on the
+Start a TCP server listening for connections on the given `port` and `host`.
+
+If `port` is omitted or is 0, the operating system will assign a random
+port, which can be retrieved by using `server.address().port`
```

Comment:
nit: Maybe use `arbitrary unused port` instead of `random port`? There doesn’t need to be any true randomness in the port choice